.github/workflows/e2e_published_vsix.yml

```diff
@@ -55,6 +55,8 @@ jobs:
     if: always()
     steps:
       - uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.PRISMA_BOT_TOKEN }}
```

Comment:
Why is it needed here?

Asking because it's also passed below in `env`

Reply:
Only one should be needed

Reply:
<img width="872" alt="Screenshot 2024-06-18 at 10 47 28" src="https://github.com/prisma/language-tools/assets/1328733/ff02d77f-659a-45a6-b244-e5d159916978">

Reply:
Both are needed:
- for checkout, it's needed because bot needs to be able commit to the repo. It has nothing to do with API calls, it's just how checkout action works
- the one below, in `env` is actually used for API requests